### ISSUE TITLE
Lisp function `part` got deleted in Alda 2

### ISSUE THREAD

**Issue by UlyssesZh** [alda-lang/alda#484]
The following plays in Alda 1 but not in Alda 2:
```
(part "piano") c d e f
```
Errors in Alda 2:
```
<no file>:1:2 unresolvable symbol: part
```
It seems that `part` function is deleted without mentioning in the migration guide.

**Comment by daveyarwood** [comment]
Good catch! I think we should add support for `part` in Alda 2 to be consistent.

**Comment by Br0wnHammer** [comment]
Hey, is this issue still open to work on? If so, I would like to contribute.

**Comment by daveyarwood** [comment]
@Br0wnHammer Yes, it's yours if you want it. 🙂

**Comment by daveyarwood** [comment]
I just released Alda 2.4.2, which includes a basic version of the `part` function. (Thanks for the PR, @CodeForBeauty!). This supports the basic, one instrument use case, e.g. `(part "piano")`, just like in Alda 1.

One caveat: the current implementation only supports declaring a single instrument at a time. More advanced part declarations — multiple instruments in a group (e.g. `trumpet/piano`) and aliases (e.g. `'trumpiano'`) — are not yet supported via the part Lisp function -- that's a TODO for the future, if anyone wants to tackle it.
